.changeset/heavy-grapes-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@vercel/blob": patch
```

Comment:
This is a minor version since this is a new feature (we use semver if you want to learn more)